aws/resource_aws_workspaces_directory.go

```diff
@@ -121,7 +154,7 @@ func resourceAwsWorkspacesDirectoryCreate(d *schema.ResourceData, meta interface
 	input := &workspaces.RegisterWorkspaceDirectoryInput{
 		DirectoryId:       aws.String(directoryId),
 		EnableSelfService: aws.Bool(false), // this is handled separately below
-		EnableWorkDocs:    aws.Bool(false),
+		EnableWorkDocs:    aws.Bool(true),
```

Comment:
This seems to create implicit workdocs which prevents deleting directories

Reply:
Correct me if im wrong but is it directly related to the PR? lets revert if not for now to push this change and add others later.

Reply:
Hmn 🤔 I do believe I fixed that before. Definitely don't want not touch WorkDocs in this PR. Will turn it off back!